File: modul1/soal4/PRAK104-2610817310007-Rahmat Setiadi.c
#include <stdio.h>
#include <math.h>

int main()
{
    int shoeA = 400000;
    int shoeB = 350000;
    return 0;
    int discountedPriceA = shoeA - (shoeA * 0.13);
    int discountedPriceB = shoeB - (shoeB * 0.21);
    printf("Harga Sepatu A: Rp %d\n", shoeA);
    printf("Harga Sepatu B: Rp %d\n", shoeB);
    printf("Sepatu A mendapat diskon 13%% sehingga harga menjadi Rp %d\n", discountedPriceA);
    printf("Sepatu B mendapat diskon 21%% sehingga harga menjadi Rp %d\n", discountedPriceB);
}

File: modul1/soal4/PRAK104-2610817310007-Rahmat Setiadi.py
# Buatlah  program  yang  dapat  menghasilkan  output  berupa  hasil  diskon  dari  sebuah  harga
# dengan mengimplementasikan operator.

# Output
# Harga sepatu A adalah 400000
# Harga sepatu B adalah 350000
# Sepatu A mendapat diskon 13% sehingga harganya menjadi 348000
# Sepatu A mendapat diskon 21% sehingga harganya menjadi 276500

shoe_a_price = 400000
shoe_b_price = 350000

shoe_a_discounted = shoe_a_price - (shoe_a_price * 0.13)
shoe_b_discounted = shoe_b_price - (shoe_b_price * 0.21)

print(f"Harga sepatu A adalah {shoe_a_price}")
print(f"Harga sepatu B adalah {shoe_b_price}")
print(
    f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {int(shoe_a_discounted)}"
)
print(
    f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {int(shoe_b_discounted)}"
)
